artman/cli/main.py

```diff
@@ -113,6 +111,14 @@ def parse_args(*args):
              '/path/to/config.yaml:config_section_A|config_section_B. '
              'This is an advanced feature; for most cases, use `--api`.',
     )
+    api.add_argument('--batch',
```

Comment:
That would be tricky to implement, because `--api` is an alias for a long `--config` argument.
Honestly, if you wanted this, it would be easier to just use a bash loop:

```bash
for API in ( pubsub bigquery vision ) ; do
  artman --api=$API --language=java --publish=github
done
```